build.nix

```diff
@@ -51,6 +51,9 @@ in stdenv.mkDerivation {
   cargoRoot = "rust-src";
   cargoDeps = rustPlatform.importCargoLock {
     lockFile = "${src}/rust-src/Cargo.lock";
+    outputHashes = {
+      "bindgen-0.64.0" = "sha256-GNG8as33HLRYJGYe0nw6qBzq86aHiGonyynEM7gaEE4=";
```

Comment:
What's the workflow going to be when this gets updated?

Reply:
Usually it's enough to run `nix build` and the build will complain about incorrect or missing hashes. However this can be also automated with nix-update (added as a comment to this file).

Reply:
This is only needed for git-based crates. Other crates can be interfered from the cargo.lock file.